### PR TITLE
Copy original file

### PR DESCRIPTION
**Pull request by ChrispaixK**
Copy original html file, css file and images folder

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<body>
+    <section id="main">
+        <h1 id="title">The Pyramids</h1>
+        <p class="cap">Impressive ancient structure</p>
+        <div id="img-div">
+            <img src="./images/pyramids-2371501-1920.jpg" id="image">
+            <caption id="img-caption">The pyramids are more than 7000 years old</caption>
+        </div>
+        <div id="tribute-info">
+            <h2 style="text-align: center;">Description</h2>
```
Comment (ChrispaixK) [inline index.html:19]: I suggest you add your CSS content to the CSS file

Comment (ChrispaixK) [inline index.html:19]: I suggest you to put your CSS content into CSS file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                </li>
+                <li>Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet.
+                    Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet.
+                </li>
+                <li>Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet.
+                    Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet.
+                </li>
+            </ul>
+        </div>
+        <p class="footer">If you have time, you should read more about this incredible human being on his <span><a href="https://www.wikipedia.org/" id="tribute-link" target="_blank">Wikipedia entry</a> </span>.</p>
```
Comment (ChrispaixK) [inline index.html:38]: Use the corresponding HTML tag for the footer

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ChrispaixK) [inline index.html:7]: I suggest you change the HTML title and make it more descriptive

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+#img-caption {
+    text-align: center;
+
+}
+
+#image {
+    width: 100%;
+}
+
+li{
```
Comment (ChrispaixK) [inline styles.css:35]: styling directly on HTML tag is considered as bad practice, you can use class or another selector instead

**Review by ChrispaixK — COMMENTED** [review]
Hello there! 
You have done an excellent job so far

##Here are some changes request before merging your work
* There is no linters add HTML and CSS file please to help you check for errors
* I think you can improve your README file and make it more descriptive
* Your project don't have a license I suggest adding one

File: index.html (hunk 1 above)
Comment (zieeco) [reply index.html:19]: **[REQUIRED CHANGES]**

- [ ] Great review here :heavy_check_mark:  As this reason is obvious to you, please kindly tell the code author why you want them to put the CSS content into a CSS file to make your review more implicit to telling the reason behind your proposed changes.

File: index.html (hunk 2 above)
Comment (zieeco) [reply index.html:38]: **[REQUIRED CHANGES]**

- [ ] Great review here :heavy_check_mark:  As this reason is obvious to you, please kindly tell the code author why you want them to use the corresponding footer tag to make your review more implicit in telling the reason behind your proposed changes.

File: styles.css (hunk 4 above)
Comment (zieeco) [reply styles.css:35]: **[REQUIRED CHANGES]**

- [ ] Great review here also :heavy_check_mark: I think you should re-evaluate your review as the code author didn't use inline styling which is contradicting your review as of now, Also, this is a CSS file. I think your reason should be not to use multiple ids on the HTML elements for styling purposes but for javascript. Please, kindly update your reasons to be in line with your review.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @ChrispaixK 


Great work on making the code review 👏 
You've done well with the reviews, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice beginning. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ChrispaixK** [comment]
Thanks for your help @zieeco

**Comment by Kakalanp** [comment]
# STATUS: APPROVED :green_circle: 

### Your project is approved @ChrispaixK! :tada: 

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :clinking_glasses: 

almost there

## Optional suggestions

Try adding some emojis to your next reviews, it'll make your review more human-friendly :alien: :robot: 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**